ui/tests/acceptance/secrets/backend/ssh/ssh-configuration-test.js

```diff
@@ -64,29 +70,61 @@ module('Acceptance | ssh | configuration', function (hooks) {
     assert.dom(SES.ssh.configureForm).exists('renders ssh configuration form');
 
     // default has generate CA checked so we just submit the form
-    await click(SES.ssh.sshInput('configure-submit'));
+    await click(SES.ssh.save);
     assert.strictEqual(
       currentURL(),
-      `/vault/secrets/${sshPath}/configuration/edit`,
-      'stays on configuration form page.'
+      `/vault/secrets/${sshPath}/configuration`,
+      'after configuring it navigates to the details page'
     );
 
-    await waitFor(SES.ssh.sshInput('public-key'));
```

Comment:
hit some local flakyness when running this multiple times so opted for a waitUntil over the old waitFor with a little longer than the default 1000ms wait time.